src/Tes.RunnerCLI/Program.cs

```diff
@@ -13,5 +13,13 @@ static async Task<int> StartUpAsync(string[] args)
     CommandFactory.CreateExecutorCommand(rootCommand);
     CommandFactory.CreateDownloadCommand(rootCommand);
 
-    return await rootCommand.InvokeAsync(args);
```

Comment:
Is this code really needed? The command invocation should already return the error code, and the exception should be thrown and logged. If was added for troubleshooting while running locally, you can change the behavior of the runner to output to the standard out instead of storage by not setting the event log sink in the task definition.

Reply:
Any exception reaching here was either already logged (before being rethrown) or was never logged (in which case this might give it more visibility in very limited circumstances) but without this change it would not have prevented a zero exit code.

Right now I'm not going to make perfect the impediment of good. If more needs to be done here, we should do it systemically and correctly as a separate issue. Please open one if needed,